index.html

```diff
@@ -19,10 +20,188 @@ <h1 class="textdeco1 text1width">José Fernández-Caminero</h1>
 
         <p class="crazyParagraph">I love Full-Stack Engineering and I can code websites that are both the most complex and the most efficient.</p>
 
-        <button>Start collaboration</button>
+        <button class="button1 width90">Start collaboration</button>
     </section>
 
-    <footer class="footerDeco">
+    <section class="middleDeco"></section>
+
+    <section>
+        <h2 class="textdeco2">Projects</h2>
+
+        <img class="fullWidth" src="resources/Rectangle 21.png" alt="Laptop and other objects">
+        <div class="blueBox">
```

Comment:
Nice job using CSS Grid for individual cards, however, according to the project requirement, it would be nice if you could use CSS Grid for the main container (`section`) and flexbox to layout individual cards, this will also help with responsiveness for larger screen size. 👍 

>  <img width="1045" alt="Screenshot 2022-10-31 at 18 06 20" src="https://user-images.githubusercontent.com/66112760/199066806-2fb60646-d77f-4c7d-b426-d50a71d0e5fb.png">

Reply:
Hi @bimbolabuari 

Thank you so much for your review. Is it ok if I use both grid and flexbox on the individual cards, or should I stick to only flexbox on the cards?